source/read/specify-a-query.txt

```diff
@@ -0,0 +1,265 @@
+      :start-after: start-find-array
+      :end-before: end-find-array
+      :language: kotlin
+      :dedent:
+   
+   .. output::
+      :visible: false
+
+      Fruit(id=1, name=apples, quantity=5, rating=3, color=red, type=[fuji, honeycrisp])
+      Fruit(id=3, name=oranges, quantity=6, rating=2, color=null, type=[naval, mandarin])
+
+Element Operators
+-----------------
+
+Element operators query data based on the presence or type of a field.
+
+To learn more about element operators, see the :manual:`Element Query Operators
+</reference/operator/query-element/>` guide in the MongoDB Server manual.
+
+The following example specifies an element operator in a query filter as a
+parameter to the ``find()`` method. The code returns all documents that have a
+``color`` field:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/read/specify-a-query.kt
+      :start-after: start-find-element
+      :end-before: end-find-element
+      :language: kotlin
+      :dedent:
+   
+   .. output::
+      :visible: false
+
+      Fruit(id=1, name=apples, quantity=5, rating=3, color=red, type=[fuji, honeycrisp])
+      Fruit(id=2, name=bananas, quantity=7, rating=4, color=yellow, type=[cavendish])
+      Fruit(id=4, name=pineapples, quantity=3, rating=5, color=yellow, type=null)
+
+Evaluation Operators
+--------------------
+
+Evaluation operators return data based on evaluations of either individual
+fields or the entire collection's documents.
+
+The following is a list of common evaluation operators:
+
+- ``$text``, which performs a text search on the documents
+- ``$regex``, which returns documents that match a specified regular expression
+- ``$mod``, which performs a modulo operation on the value of a field and
+  returns documents where the remainder is a specified value
+
+To view a full list of evaluation operators, see the :manual:`Evaluation Query Operators
+</reference/operator/query-evaluation/>` guide in the MongoDB Server manual.
+
+The following example specifies an evaluation operator in a query filter as a
+parameter to the ``find()`` method. The code uses a regular expression to return
+all documents with a ``name`` field value that has at least two consecutive
+``"p"`` characters:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/read/specify-a-query.kt
+      :start-after: start-find-evaluation
+      :end-before: end-find-evaluation
+      :language: kotlin
+      :dedent:
+   
+   .. output::
+      :visible: false
+
+      Fruit(id=1, name=apples, quantity=5, rating=3, color=red, type=[fuji, honeycrisp])
+      Fruit(id=4, name=pineapples, quantity=3, rating=5, color=yellow, type=null)
+
+Additional Information
+----------------------
+
+To learn more about querying documents, see the :manual:`Query Documents
+</tutorial/query-documents/>` guide in the MongoDB Server manual.
+
+To learn more about retrieving documents with the {+driver-short+}, see
+:ref:`kotlin-sync-retrieve`.
```

Comment:
Note for reviewers: The retrieve page here is currently also in review. This PR should be merged in after the retrieve page is merged.

[Link to Retrieve page PR](https://github.com/mongodb/docs-kotlin-sync/pull/6)